include/mapbox/geojson.hpp

```diff
@@ -40,3 +40,7 @@ std::string stringify(const geojson &);
 
 } // namespace geojson
 } // namespace mapbox
+
+#if !defined(MAPBOX_GEOJSON_LIBRARY)
+#include "geojson_impl.hpp"
+#endif
```

Comment:
I think we should not do this in a header file, but rather in a separate `*.cpp` file to avoid issues with having `geojson.hpp` included more than once, which leads to duplicate definitions. So instead of including the implementation here, we should do it in `src/mapbox/geojson.cpp`.